index.html

```diff
@@ -1429,6 +1429,25 @@ <h4>Rate obfuscation</h4>
           received from the platform collector during this penalty.
         </p>
       </section>
+      <section>
+        <h4>Rate obfuscation parameters</h4>
+        <p><i>This section is non-normative.</i></p>
+        <p>
+          Based on implementation experience, implementers are advised to use:
+          <ul>
+            <li>
+              a range in between 300000 milliseconds (5 minutes) and 600000 milliseconds (10 minutes) for |observer|.{{PressureObserver/[[ObservationWindow]]}}.
```

Comment:
Editorial nitpick: the `|variable|` notation is generally used in algorithms. `observer` has not been defined here. I'd just say something along the lines of "a range [...] for PressureObserver's [[ObservationWindow]] internal slot".

Reply:
agree!